src/components/main/flight/FlightInfoScreen.js

```diff
@@ -239,265 +353,355 @@ export default class FlightInfoScreen extends React.Component {
   },
-  loadingText: {
-    marginTop: '5%',
-    fontFamily: 'Montserrat-bold',
-    fontSize: 12,
-    color: COLORS.lightblue,
-    textAlign: 'center'
+  backDrop: {
+    flex: 1,
+    backgroundColor: "transparent",
   },
-  container: {
+  // Content Wrapper
+  innerView: {
     flex: 1,
-    flexDirection: 'column',
-    backgroundColor: COLORS.white,
-    height: height,
-    width: width
+    flexDirection: "column",
+    marginLeft: Math.round(moderateScale(105, 0.625)),
+    marginRight: Math.round(moderateScale(20, 0.0625)),
+    marginBottom: Math.round(moderateScale(30, 0.0625)),
+    marginTop: Math.round(moderateScale(80, 0.0625)),
+  },
+
+  // Top Inner View
+  topInnerView: {
+    flexDirection: "column",
+    marginBottom: Math.round(verticalScale(30)),
+    paddingBottom: Math.round(verticalScale(10)),
   },
-  containerTop: {
-    paddingLeft: width * 0.05,
-    paddingRight: width * 0.05,
-    paddingTop: height * 0.06,
-    marginBottom: height * 0.10,
-    backgroundColor: COLORS.white,
+  flightView: {
+    marginBottom: Math.round(verticalScale(20)),
   },
-  buttonBarNav: {
-    flexDirection: 'row',
-    height: height * 0.05,
-    justifyContent: 'space-between',
-    marginBottom: height * 0.05,
+  flightNumberLabel: {
+    color: COLORS.lightSandy,
+    fontFamily: "Poppins",
+    fontSize: Math.round(moderateScale(16, 0.0625)),
   },
-  buttonBarTop: {
-    flexDirection: 'row',
-    height: height * 0.04,
-    justifyContent: 'center',
-    width: '66%',
+  flightNumberText: {
+    color: COLORS.lightSandy,
+    fontFamily: "Poppins-bold",
+    fontSize: Math.round(moderateScale(40, 0.0625)),
   },
-  buttonBarBottom: {
-    flexDirection: 'row',
-    height: height * 0.04,
-    justifyContent: 'center',
+  flightInfoText: {
+    color: COLORS.lightSandy,
+    fontFamily: "Poppins",
+    fontSize: Math.round(moderateScale(16, 0.0625)),
   },
-  leftGreenButton: {
-    borderTopLeftRadius: 5,
-    borderBottomLeftRadius: 5,
-    marginRight: 3,
-    backgroundColor: COLORS.lightgreen,
+
+  // itinerary View
+  itineraryView: {
+    flexDirection: "row",
+    alignItems: "center",
+  },
+  plane_icon_view: {
     flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
   },
-  middleGreenButton: {
-    backgroundColor: COLORS.lightgreen,
-    marginRight: 3,
+  plane_icon: {
     flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
+    color: COLORS.lightSandy,
+    textAlign: "center",
+    fontSize: Math.round(moderateScale(35, 0.0625)),
   },
-  rightGreenButton: {
-    borderTopRightRadius: 5,
-    borderBottomRightRadius: 5,
-    backgroundColor: COLORS.lightgreen,
+  itineraryViewItem: {
     flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
+    flexDirection: "column",
   },
-  planeInfoText: {
-    paddingBottom: height * 0.03,
+  itineraryLabel: {
+    color: COLORS.lightSandy,
+    textAlign: "center",
+    fontSize: Math.round(moderateScale(14, 0.0625)),
```

Comment:
let's shrink the base font size down to 10 or 12

Reply:
Unless you want to scale the entire UI down, decreasing this label makes it look SUPER small in contrast.

Reply:
Well San Francisco International used to fit on 2 lines, now it hits 3.

Reply:
I am looking at it right now and it's 2 lines. Dude, can I be real, that's how it works on your device. We need to prioritize a specific set of devices rather than the an entire array of them. The number of lines was something that would change anyway based on the length of the flight name.

Reply:
So we want to write off anyone using a samsung galaxy Note? which is one of the most common android phones in the US?

Reply:
I'm just saying, it used to be 2 lines, what changed to make it 3?

Reply:
I am resolving this. Tbh, I get that you are a perfectionist, but this, "this is big but doesn't break anything," "this is too small but doesn't break anything,"  style of reviewing is not  working for me.

Reply:
Cool. If you feel like this is what you want to submit for this screen that's fine. I'll pick it up and work on it after my release at work and I have some downtime.